### PR TITLE
#61 Icons Doc

### PR DESCRIPTION
**Pull request by kevinmpowell**
@nathanacurtis and @jamesmelzer this doc is ready for review. Please follow the instructions to compile the tokens locally and confirm it works on your machines.

References: https://github.com/EightShapes/esds-library/issues/61

### CODE REVIEW

**Comment by jamesmelzer** [comment]
After successful `npm install` and `lerna bootstrap` in this branch, all the other commands in the readme failed with `missing script [command name]`. 

Doc Feedback

- where things live: clear
- build commands: clear
- adding new assets: clear
- injecting icons into markup: clear
- dependency rationale: clear

missing: Which of the build commands are run as part of the `default` listener (is that npm run dev or npm start or similar)? Or if none of them run automatically, that might be worth stating, e.g. "For performance reasons, icon build commands should be run manually and are not included in the default listener [command]" or similar.

**Comment by jamesmelzer** [comment]
@kevinmpowell Checking myself: I didn't run npm commands in the correct folder, esds-icons, which totally explains why the commands didn't work.

File: esds-icons/README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+```
+Serves as the `main` output of this package. Contains ES6 import/export statements for each module.
+
+### Icon Names JSON
+```
+dist/esds-icon-names.json
+```
+A JSON array containing individual SVG filenames minus the extension. Useful for iterating over the icon set programmaticaly, to generate an icon gallery.
+
+##Building Icon Outputs
```
Comment (nathanacurtis) [inline esds-icons/README.md]: Need a space after heading.

File: esds-icons/README.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+import { EsdsIconCaretDown, EsdsIconCaretUp } from '@eightshapes/esds-icons';
+```
+
+
+
+## Icon Gallery
+<!-- MARKDOWNGALLERY: DO NOT EDIT THIS SECTION, AUTOGENERATED -->
+||||||||
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+|<img height="80" width="80" src="src/angle-left.svg" alt="angle-left"/><br>angle-left|<img height="80" width="80" src="src/angle-right.svg" alt="angle-right"/><br>angle-right|<img height="80" width="80" src="src/bars.svg" alt="bars"/><br>bars|<img height="80" width="80" src="src/caret-down.svg" alt="caret-down"/><br>caret-down|<img height="80" width="80" src="src/caret-up.svg" alt="caret-up"/><br>caret-up|<img height="80" width="80" src="src/check.svg" alt="check"/><br>check|<img height="80" width="80" src="src/eightshapes-logo.svg" alt="eightshapes-logo"/><br>eightshapes-logo|
```
Comment (nathanacurtis) [inline esds-icons/README.md]: Please remove the logo and mark from the icon set. They should be a separate component.

**Pull request merged**